Scripts/Player_Scripts/player.gd

```diff
@@ -0,0 +1,37 @@
+extends CharacterBody3D
+
+
+@export var SPEED = 5.0
```

Comment:
all upper case is for constants.

These are exports, so not constant. They should be snake cases.